pkg/accounting/db.go

```diff
@@ -36,7 +37,35 @@ type Rollup struct {
 	AtRestTotal    float64
 }
 
-// DB stores information about bandwidth usage
+// BucketBWRollup mirrors dbx.BucketBandwidthRollup and holds the bucket bw rollup info
```

Comment:
Avoid using shorthands they usually reduce legibility. If the name seems too long, this usually indicates that there's a nicer way to organize code.